packages/property-grid/src/components/ElementList.scss

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@bentley/ui-core/lib/ui-core/style/index";
+@import "~@bentley/ui-core/lib/ui-core/text/index";
+@import "@bentley/ui-core/lib/ui-core/index";
```

Comment:
missing the `~` in import for line 7. If you import the barrel on line 7, i dont think you need the ones on 5 and 6

Reply:
Fixed.